<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Pragti" failures="3" tests="3" name="Default test(failed)" time="0.372" errors="0" timestamp="2024-06-13T15:42:08 IST">
  <testcase classname="alwayspracticehere.OnlyfailedtcexecuteTest" name="fkkTest" time="0.129">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at alwayspracticehere.OnlyfailedtcexecuteTest.fkkTest(OnlyfailedtcexecuteTest.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- fkkTest -->
  <testcase classname="alwayspracticehere.OnlyfailedtcexecuteTest" name="ufTest" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [8] but found [5]">
      <![CDATA[java.lang.AssertionError: expected [8] but found [5]
at alwayspracticehere.OnlyfailedtcexecuteTest.ufTest(OnlyfailedtcexecuteTest.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- ufTest -->
  <testcase classname="alwayspracticehere.OnlyfailedtcexecuteTest" name="fTest" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [8] but found [5]">
      <![CDATA[java.lang.AssertionError: expected [8] but found [5]
at alwayspracticehere.OnlyfailedtcexecuteTest.fTest(OnlyfailedtcexecuteTest.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- fTest -->
  <testcase classname="alwayspracticehere.OnlyfailedtcexecuteTest" name="foTest" time="0.0">
    <ignored/>
  </testcase> <!-- foTest -->
</testsuite> <!-- Default test(failed) -->
